### PR TITLE
Update ember-cli, add Travis badge and some other tweaks

### PR DESCRIPTION
**Pull request by vladucu**
### What's here

- updates to latest ember-cli and fam
- fix / tweak some tests
- add travis badge to readme
- configure ember-try to use `yarn`
- update `repository` in `package.json`, hoping to fix `ember-polaris` not being on emberobserver/emberaddons

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -43,15 +45,6 @@ module.exports = {
...
       })
-    },
-
-    // test files
-    {
-      files: ['tests/**/*.js'],
-      excludedFiles: ['tests/dummy/**/*.js'],
-      env: {
-        embertest: true
-      }
```
Comment (andrewpye) [inline .eslintrc.js]: This doesn't mean that linting's been removed from our test files does it? 😬

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +24,7 @@ env:
     # as well as latest stable release (bonus points to beta/canary)
     - EMBER_TRY_SCENARIO=ember-lts-2.12
     - EMBER_TRY_SCENARIO=ember-lts-2.16
+    - EMBER_TRY_SCENARIO=ember-lts-2.18
```
Comment (andrewpye) [inline .travis.yml]: 👍

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# ember-polaris
+ember-polaris
+==============================================================================
+
+[![Build Status](https://travis-ci.org/smile-io/ember-polaris.svg?branch=master)](https://travis-ci.org/smile-io/ember-polaris)
```
Comment (andrewpye) [inline README.md]: 🙌

File: README.md [hunk 4]
```diff
@@ -8,12 +11,13 @@
...
 
-## Installation
+Installation
+------------------------------------------------------------------------------
 
 Install `ember-polaris` using `ember-cli`:
 
 ```sh
-$ ember install ember-polaris
+$ ember install @smile-io/ember-polaris
```
Comment (andrewpye) [inline README.md]: 👍

File: tests/integration/components/polaris-page-test.js [hunk 5]
```diff
@@ -250,7 +250,7 @@ test('it renders action icons correctly', function(assert) {
   assert.equal(secondaryActionIcons[1].dataset.iconSource, 'polaris/cancel', 'second secondary action icon - renders the correct icon');
 });
 
-test('it handles breadcrumbs correctly', function(assert) {
+skip('it handles breadcrumbs correctly', function(assert) {
```
Comment (andrewpye) [inline tests/integration/components/polaris-page-test.js]: Why are we skipping this?!

File: tests/integration/components/polaris-thumbnail-test.js [hunk 6]
```diff
@@ -36,8 +36,7 @@ test('it renders the correct HTML when all attributes are passed in', function(a
 
 
 test('it renders a correctly-sized thumbnail', function(assert) {
-  this.set('size', null);
```
Comment (andrewpye) [inline tests/integration/components/polaris-thumbnail-test.js]: Any reason for explicitly getting rid of this `null`?

File: .eslintrc.js (hunk 1 above)
Comment (vladucu) [reply .eslintrc.js]: no no

File: tests/integration/components/polaris-thumbnail-test.js (hunk 6 above)
Comment (vladucu) [reply tests/integration/components/polaris-thumbnail-test.js]: it's kind of redundant.....we pass 'unsupported' at the end and test basically exactly this

File: tests/integration/components/polaris-page-test.js (hunk 5 above)
Comment (vladucu) [reply tests/integration/components/polaris-page-test.js]: oh, this is my bad...will revert

File: tests/integration/components/polaris-thumbnail-test.js (hunk 6 above)
Comment (andrewpye) [reply tests/integration/components/polaris-thumbnail-test.js]: `null` is different to a string that isn't a member of an enum... 🤷‍♂️ but OK 😛

Comment (vladucu) [reply tests/integration/components/polaris-thumbnail-test.js]: wait...why is it different? it's an enum....`null` like `unsupported` is a value that's not in the enum....shouldn't it be exact same thing?

**Comment by vladucu** [comment]
k, gonna revert that skipped test and ship this if it's ok with you

File: tests/integration/components/polaris-thumbnail-test.js (hunk 6 above)
Comment (andrewpye) [reply tests/integration/components/polaris-thumbnail-test.js]: The outside world doesn't know how `size` is being used internally - the component's a black box. Just thought it was worth testing that it doesn't blow up if it's given `null` - imagine if to perform a case-insensitive check against the enum, the component was doing `size.toLowerCase()` for example - `'unsupported'` would be fine, but `null` would blow up

**Comment by andrewpye** [comment]
Yeah go for it 🙌

File: .eslintrc.js (hunk 1 above)
Comment (vladucu) [reply .eslintrc.js]: yeah, this is just from the ember update, not our changes

Comment (andrewpye) [reply .eslintrc.js]: OK cool, thanks!

File: addon/components/polaris-breadcrumbs/breadcrumb.js [hunk 7]
```diff
@@ -14,7 +14,7 @@ export default LinkComponent.extend({
 
   breadcrumb: null,
 
-  params: computed('breadcrumb.{content,route,models.@each.id}', function() {
+  params: computed('breadcrumb.{content,route,models.[]}', function() {
```
Comment (vladucu) [inline addon/components/polaris-breadcrumbs/breadcrumb.js]: @andrewpye fyi, forgot about it, this was the reason the test was skipped (it was failing)....fixed with this loosened dep
screen shot 2018-05-18 at 2 38 24 pm

Comment (andrewpye) [reply addon/components/polaris-breadcrumbs/breadcrumb.js]: Thanks for the heads-up @vladucu! Weird one, any idea what changed to cause this?

My thinking would be more that this was a test setup issue, but I'm happy with your fix too 👍

**Pull request merged**